server/wizard/steps/step_greeting.go

```diff
@@ -0,0 +1,31 @@
+package steps
+
+import (
+	"github.com/mattermost/mattermost-plugin-api/experimental/flow"
+)
+
+const (
+	stepNameGreeting = "greeting"
+
+	stepTitleGreeting = ""
+
+	stepDescriptionGreeting = `:wave: Welcome to Zoom for Mattermost! I'll walk you through the process here, so you can begin using the integration. Feel free to read the [documentation](https://mattermost.gitbook.io/plugin-zoom) if you'd like.
```

Comment:
Should the introduction text be similar to the other plugin as first introduce the steps that will be taken?
![Screenshot from 2022-02-25 12-45-15](https://user-images.githubusercontent.com/16541325/155709945-aca386c9-bc37-4958-8adb-566c8a06fdcc.png)